src/commands/create.js

```diff
@@ -112,34 +117,36 @@ export default asyncCommand({
 
 		spinner.text = 'Installing dev dependencies';
```

Comment:
Oh, I misread the diff last time. Yeah, ^^ what I was trying to say last time. 😴